pkg/resources/validation/cluster/provisioningcluster.go

```diff
@@ -164,6 +178,93 @@ func (p *ProvisioningClusterValidator) validateClusterName(request *admission.Re
 	return nil
 }
 
+// validatePSACT validate if the cluster and underlying secret are configured properly when PSACT is used in or dropped from the cluster
```

Comment:
Suggestion: `validates if the cluster and underlying secret are configured properly when PSACT is enabled or disabled`

Reply:
fixed